realm/realm-library/src/objectServer/java/io/realm/ErrorCode.java

```diff
@@ -127,8 +127,8 @@ public static ErrorCode fromInt(int errorCode) {
         throw new IllegalArgumentException("Unknown error code: " + errorCode);
     }
 
-public enum Category {
+    public enum Category {
         FATAL,          // Abort session as soon as possible
-        RECOVERABLE    // Still possible to recover the session by either rebinding or providing the required information.
+        RECOVERABLE,    // Still possible to recover the session by either rebinding or providing the required information.
```

Comment:
https://docs.oracle.com/javase/specs/jls/se7/html/jls-8.html#jls-8.9

Java language spec says that enum accepts optional `,` before EnumBodyDeclarations.